### DISCUSSION TITLE
2.0版本doris使用spark写入，并行度参数不生效

### DISCUSSION THREAD

**Discussion by HK2009** [apache/doris#39130] [A - General / Q&A]
设置的参数为：
.option("doris.sink.task.use.repartition", "true")
.option("doris.sink.task.partition.size", "1")

但doris的合并压力太大，还是产生了太多次streamload请求，收到报错。
detailMessage = current running txns on db 11341 is 1000, larger than limit 1000

**Comment by maruyue** [comment]
建议英文提问，如使用中文提问，可以去ask.selectdb.com中文技术论坛。

**Comment by trunckman** [comment]
我看过1.2版本的源码，spark写入并行度由bucket数量决定；
如果spark写入单分区的bucket数量是1 ，无论怎么调spark参数，并行度还是1
